src/components/GoodList/GoodList.jsx

```diff
@@ -0,0 +1,11 @@
+import { GoodInfo } from '../GoodInfo';
+
+export const GoodList = ({ goods }) => (
+  <>
+    <ul>
+      {goods?.map(good => (
+        <GoodInfo good={good} />
```

Comment:
```suggestion
    <ul>
      {goods?.map(good => (
        <GoodInfo good={good} />
```
Key attribute is missing